sway-lib-std/src/error_signals.sw

```diff
@@ -3,5 +3,9 @@ library error_signals;
 
 /// revert with this value for a failing call to std::revert::require.
 pub const FAILED_REQUIRE_SIGNAL = 42;
+
 /// revert with this value for a failing call to std::message::send_message.
 pub const FAILED_SEND_MESSAGE_SIGNAL = 0xffff_ffff_ffff_0002;
+
+/// revert with this value for a failing call to std::token::transfer_to_address.
+pub const FAILED_TRANSFER_TO_ADDRESS_SIGNAL = 0xffff_ffff_ffff_0001;
```

Comment:
Do you mind keeping these in numerical order? Eventually `FAILED_REQUIRE_SIGNAL` should be changed to `0xffff_ffff_ffff_0000` so we can keep it at the start.

Reply:
just made the change to  `FAILED_REQUIRE_SIGNAL` and ordered them numerically!